meerkat_hermes/config.py

```diff
@@ -28,7 +28,7 @@ class Config(object):
         'Notifications <notifications@emro.info>'
     )
 
-    PUBLISH_RATE_LIMIT = int(os.environ.get("MESSAGE RATE LIMIT", "75"))
+    PUBLISH_RATE_LIMIT = int(os.environ.get("MESSAGE_RATE_LIMIT", "100"))
```

Comment:
What was the reasoning behind this?  It's potentially fine, just interested if you actually had a problem with sending more than a message a minute...